README.md

```diff
@@ -82,6 +82,15 @@ Below is a table showing ``relative_start_pos`` (`start`) and ``relative_end_pos
 | ``<i> ... </i><b></b>`` | 0 | 0 | 1 | 2 |
 | ``<i> ... <b></b></i>`` | 0 | 2 | 0 | 1 |
 
+## Pre-processing Metadata:
+
+### Entity Tags.
+
+**Pre-requisite steps for preprocessing Entity Tags**
+
+* Run `pip install -r bsmetadata/vendor/REL/requirements.txt`
+* Run `preprocessing_scripts/download_entity_processing_files.sh` to download all the required files.
```

Comment:
I guess that currently the first step is to run the `git submodule update --init --recursive` command ? If so, it would be nice to add it to the readme (this addition will not be necessary if we go with the `extras_require` solution describe in the comment above :wink: )